scss/common/_base.scss

```diff
@@ -540,7 +540,7 @@
         content: "";
         box-sizing: inherit;
         border-radius: 50%;
-        border-width: .03em;
+        border-width: .03125em;
```

Comment:
This value should be 0.05 as per https://github.com/telerik/kendo-theme-default/pull/768